modules/emoji.js

```diff
@@ -0,0 +1,11 @@
+function trigger(emoji, api, message) {
+  var threadID = message.threadID;
+  // TODO: validate facebook friendly emojis so sadbot can give an error
```

Comment:
If the emoji specified isn't one of the ones facebook likes, it just does nothing. We should catch this so if someone says `/prefs emoji smiley` we don't wipe out the valid firebase settting. see #55